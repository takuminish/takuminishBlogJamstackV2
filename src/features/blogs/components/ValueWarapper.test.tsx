import React from "react";
import { render } from "@testing-library/react";
import ValueWrapper from "./ValueWarapper";

const createDOM = () => render(<ValueWrapper>test</ValueWrapper>);

test("snapshot", () => {
  const { container } = createDOM();

  expect(container).toMatchSnapshot();
});
