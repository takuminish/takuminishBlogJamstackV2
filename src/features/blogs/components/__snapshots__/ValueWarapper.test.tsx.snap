// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-l57rop"
  >
    test
  </div>
</div>
`;
